Netcracker/KubeMarine
# [CPDEV-91618] update public registry in documentation

### Description
Part of [PR-491](https://github.com/Netcracker/KubeMarine/pull/491) - need to update [documentation](https://github.com/Netcracker/KubeMarine/blob/main/documentation/) because we changed the default registry to [registry.k8s.io](https://github.com/Netcracker/KubeMarine/pull/491/files#diff-39319040969c955b67cc5e18dad37e9d99e60833cd002e16df8ea1fde8b0fafaR542)

### Solution
Update [documentation](https://github.com/Netcracker/KubeMarine/blob/main/documentation)


### How to apply
* Not required

### Test Cases
* Install public cluster
* Covered during [PR-491](https://github.com/Netcracker/KubeMarine/pull/491) testing 

**TestCase 1**

Test Configuration:

- Hardware: Any
- OS: Any
- Inventory: Any

Steps:

1. Install cluster without registy in inventory
2. Check registry in /etc/kubernetes/prepull-config.yaml

Results:

| Before | After |
| ------ | ------ |
| k8s.gcr.io | registry.k8s.io |


### Checklist
